acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuValidatorNode.java

```diff
@@ -205,11 +205,9 @@ public TekuValidatorNode.Config withBeaconNodes(final TekuNode... beaconNodes) {
       return this;
     }
 
-    public TekuValidatorNode.Config withBeaconNodeEventStreamSyncingStatusQueryPeriod(
-        final Duration syncingStatusQueryPeriod) {
-      configMap.put(
-          "Xbeacon-node-event-stream-syncing-status-query-period",
-          syncingStatusQueryPeriod.toMillis());
+    public TekuValidatorNode.Config withBeaconNodesSyncingQueryPeriod(
+        final Duration syncingQueryPeriod) {
+      configMap.put("Xbeacon-nodes-syncing-query-period", syncingQueryPeriod.toSeconds());
```

Comment:
Theres probably very sub-optimal times to be deciding to change back from failover back to primary, so we likely want to be careful...

If we're about to calculate duties or perform a duty, failing back to primary may result in inactivity. I'm not sure how we reconcile that... potentially we want to wait until duties are complete for a slot rather than doing it every N period of time...  I could forsee 12 / 6 / 4 / 3 all potentialy being very disruptive and leading to hard to track down inactivities.
block production is probably worse in bellatrix, now that we might be getting a blinded block and not having the EL payload, failover during that period would result in loss of block production which would be really not great...
I'm not sure what the answer would be other than checking it around half way through a slot - potentially would miss aggregation, but shouldnt then miss any attestation or block production duties (i think they start in the last couple of seconds of the slot so we'd want to be stable by then or queued behind the changeover)